src/workerd/api/node/util.h

```diff
@@ -259,6 +261,8 @@ class UtilModule final: public jsg::Object {
     JSG_METHOD(getConstructorName);
     JSG_METHOD(getCallSite);
 
+    JSG_METHOD(getEnvObject);
```

Comment:
I think we should move this implementation to a new file preferably called process. This util is node:util, and I'm afraid this file can easily become a place where we dump random functions.

Reply:
Can you at least add a TODO so we don't forget?